README.md

```diff
@@ -105,6 +105,26 @@ the next 100 are measured for inference throughput.
 The input parameters to the test are used to generate permutations
 of tests with varying configurations.
 
+### The model
+As Neural Network, we use Pytorch's implementation of Resnet50. The script `imagenet/model_saver.py`
+can be used to generate the model for CPU or GPU. It can be invoked with
+
+```bash
+python model_saver.py
+```
```

Comment:
When running `python model_saver.py` or `python model_saver_cpu.py` I receive the error `AssertionError: Torch not compiled with CUDA enabled`, I followed the installation steps from the README.md and stopped at generating the resnet file, Ill include the stack trace below

`(myenv2) richaama@horizon:/lus/cls01029/richaama/smartsim-scaling/SmartSim-Scaling/imagenet> python model_saver.py --device=CPU
USING CUDA False
Traceback (most recent call last):
  File "/lus/cls01029/richaama/smartsim-scaling/SmartSim-Scaling/imagenet/model_saver.py", line 6, in <module>
    model.cuda()
  File "/lus/scratch/richaama/miniconda3/envs/myenv2/lib/python3.9/site-packages/torch/nn/modules/module.py", line 688, in cuda
    return self._apply(lambda t: t.cuda(device))
  File "/lus/scratch/richaama/miniconda3/envs/myenv2/lib/python3.9/site-packages/torch/nn/modules/module.py", line 578, in _apply
    module._apply(fn)
  File "/lus/scratch/richaama/miniconda3/envs/myenv2/lib/python3.9/site-packages/torch/nn/modules/module.py", line 601, in _apply
    param_applied = fn(param)
  File "/lus/scratch/richaama/miniconda3/envs/myenv2/lib/python3.9/site-packages/torch/nn/modules/module.py", line 688, in <lambda>
    return self._apply(lambda t: t.cuda(device))
  File "/lus/scratch/richaama/miniconda3/envs/myenv2/lib/python3.9/site-packages/torch/cuda/__init__.py", line 210, in _lazy_init
    raise AssertionError("Torch not compiled with CUDA enabled")
AssertionError: Torch not compiled with CUDA enabled`

Reply:
OK, I think I fixed that.